spec/features/idv/doc_auth/getting_started_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.feature 'getting started step' do
+  include IdvHelper
+  include DocAuthHelper
+
+  let(:fake_analytics) { FakeAnalytics.new }
+  let(:maintenance_window) { [] }
+  let(:sp_name) { 'Test SP' }
+
+  before do
+    allow_any_instance_of(ApplicationController).to receive(:analytics).and_return(fake_analytics)
+    allow_any_instance_of(ServiceProviderSessionDecorator).to receive(:sp_name).and_return(sp_name)
+    allow_any_instance_of(Idv::WelcomeController).to receive(:getting_started_a_b_test_bucket).
```

Comment:
This works; another option is to override the bucket config to be 100% new and 0% old. That way you don't have to use the dreaded `allow_any_instance_of` method.

Reply:
The problem is that those constants get set up at load time. I did just see a `reload_ab_test_initializer!` that's defined in a couple of specs that I could move out to a helper.

Reply:
Slick, thanks!